eng/config/rulesets/Common.globalconfig

```diff
@@ -0,0 +1,99 @@
+is_global = true
+
+dotnet_diagnostic.CA1067.severity = warning
```

Comment:
Note: I did not validate that each of the entries in the global configs map to the entries in current rulesets.